lib/project_types/extension/tasks/configure_options.rb

```diff
@@ -13,7 +13,8 @@ def call(specification_attribute_sets)
       private
 
       def configure_skip_build(attributes)
-        attributes[:options].merge!(skip_build: attributes[:identifier] == "theme_app_extension")
+        attributes[:options].merge!(skip_build: attributes[:identifier] == "theme_app_extension" ||
+        attributes[:identifier] == "beacon_extension")
```

Comment:
I think in production we actually want a build step to compile the JS inputs so perhaps remove this code for now? Right @antoinegrant

Reply:
Yeah we'll need to a compile step to validate the extension and to read the extension from a module export so that we can wrap it with our closure.

Reply:
Ok then maybe we keep this line here temporarily for development purposes but remove it once we are ready to incorporate the module export